doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
+**********************************************
+
+We also considered allowing child's fields to be initialized based on
+parent's fields. For example, the child could control where, in its Phase 1,
+the parent's Phase 1 is invoked, not necessarily as the last step.
```

Comment:
There are a lot of commas in this sentence, making the thought process feel very choppy and interfering with the clarity of the thought.  I would prefer:
"For example, the child could control the point during its Phase 1 when the parent's Phase 1 would be invoked, rather than forcing this invocation to occur only as the last step."